src/hooks/rems.ordersign.ts

```diff
@@ -282,20 +283,6 @@ function buildErrorCard(reason: string) {
 }
 
 const handler = (req: TypedRequestBody, res: any) => {
-  function getFhirResource(token: string) {
-    const ehrUrl = `${req.body.fhirServer}/${token}`;
-    const access_token = req.body.fhirAuthorization?.access_token;
-    const options = {
-      method: 'GET',
-      headers: {
-        Authorization: `Bearer ${access_token}`
-      }
-    };
-    const response = axios(ehrUrl, options);
-    return response.then(e => {
-      return e.data;
-    });
-  }
   function handleCard(hydratedPrefetch: OrderSignPrefetch) {
     const context = req.body.context;
     const contextRequest = context.draftOrders?.entry?.[0].resource;
```

Comment:
Should we get the request out of the draft orders here as well since the prefetch doesn't necessarily have the request in it? It is not part of the prefetch template.

Reply:
Oh yeah, didn't read this line a few lines up from where I was trying to comment and it wouldn't let me. This seems fine then.